src/rpc/rawtransaction.cpp

```diff
@@ -1030,19 +1030,24 @@ static UniValue sendrawtransaction(const JSONRPCRequest& request)
             + HelpExampleRpc("sendrawtransaction", "\"signedhex\"")
         );
 
-    std::promise<void> promise;
```

Comment:
Currently in 745c540d05f4a21810eccc5e9450e9b321560560, wrong commit?

Reply:
I think it's the right commit? It moves into BroadcastTransaction.